Vwitter/UserCell.h

```diff
@@ -0,0 +1,20 @@
+//
+//  UserCell.h
+//  Vwitter
+//
+//  Created by Christina Li on 7/11/22.
+//
+
+#import <UIKit/UIKit.h>
+#import <Parse/Parse.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface UserCell : UITableViewCell
+@property (strong, nonatomic) PFUser *user;
```

Comment:
This is very confusing, in some places, you are using `PFUser`, but in other places, `VWUser`, I would recommend avoiding `PFUser` everywhere, since it is a subclass of `PFUser`, `VWUser` can be used everywhere `PFUser` is used.